docs/app/Examples/elements/Icon/IconSet/IconSetExamples.js

```diff
@@ -4,27 +4,29 @@ import _ from 'lodash'
 import React, { Component } from 'react'
 import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
 import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
-import { iconCategories as categories } from 'src/utils/semanticUtils'
+import { iconsByCategory as categories } from '../../../../utils/icons'
```

Comment:
The docs and test directories have root require resolve via webpack.  So, you can import `src/utils/icons` here.

We have to use relative requires in the `src` directory since user's can build that themselves if they want and we can't force them to set this up.